From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 01 Oct 2017 15:39:34 +0100
Subject: tools/lib/lockdep: Define pr_cont()

lockdep.c now also uses pr_cont(), so we need to implement it in
liblockdep.

It is currently always used to continue warning lines, so define
pr_cont() the same as pr_warn().  If this changes, we might need to
record the last log level in a TLS variable and have pr_cont() check
that.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
--- a/tools/include/linux/lockdep.h
+++ b/tools/include/linux/lockdep.h
@@ -47,6 +47,8 @@ static inline int debug_locks_off(void)
 #define printk(...) dprintf(STDOUT_FILENO, __VA_ARGS__)
 #define pr_err(format, ...) fprintf (stderr, format, ## __VA_ARGS__)
 #define pr_warn pr_err
+/* XXX we assume pr_cont() is only used for warnings */
+#define pr_cont pr_warn
 
 #define list_del_rcu list_del
 
